apps/templates/jinja/xss.html

```diff
@@ -0,0 +1,34 @@
+{% block content %}
+<h1>XSS</h1>
+<div class="column is-8">
+    <p>Any input entered will be displayed on this page</p>
+    <p>Example:
+        <xmp><img src="nonexistent-file.png" onerror=alert(document.cookie);></xmp>
```

Comment:
I had to look up `<xmp>` and it looks like it's deprecated. I'm not sure how much this sort of thing matters to us (or in general), but maybe this could be `<code>` instead. Up to you, I'm good either way